webapi/src/test/scala/org/knora/webapi/responders/v2/OntologyResponderV2Spec.scala

```diff
@@ -131,6 +130,52 @@ class OntologyResponderV2Spec extends CoreSpec() with ImplicitSender {
             }
         }
 
+        "delete the 'foo' ontology" in {
+            actorUnderTest ! DeleteOntologyRequestV2(
+                ontologyIri = fooIri.get.toSmartIri,
+                lastModificationDate = fooLastModDate,
+                apiRequestID = UUID.randomUUID,
+                requestingUser = imagesUser
+            )
+
+            expectMsgType[SuccessResponseV2](timeout)
```

Comment:
Could you maybe add another request that checks that the ontology is really not there anymore? This test would also pass if the `DeleteOntologyRequestV2` doesn't do anything besides returning a positive result.

Reply:
- maybe also a quick test that tries to delete a non-existing ontology?
- maybe also test with a project-admin user and a non-project-admin user? I suppose that only a project admin should be able to delete an ontology?